src/screens/UnlockScreen.tsx

```diff
@@ -42,16 +59,50 @@ export class UnlockScreen extends PureComponent<Props, State> {
     }
   };
 
+  handleFailedAttempt = (increasedFailedTimes: number) => {
+    const { attempt } = this.props.timeCounter;
+    const isFinalAttempt = increasedFailedTimes > 2;
+    const finalAttempt = 3;
+    const firstAttempt = 0;
+    let currentDate = dayjs();
+    let blockedTimeInMinutes = 1;
+
+    if (attempt === 0) {
+      currentDate = currentDate.add(1, 'minute');
```

Comment:
`currentDate = currentDate.add(1, 'minute')` -> `currentDate.add(blockedTimeInMinutes, 'minute')`;
you don't have to assign the value again. Use the blockedTimeInMinutes